### PR TITLE
implemented Prisma installation according to the user

### PR DESCRIPTION
**Pull request by sooraj1002**

### CODE REVIEW

**Comment by sooraj1002** [comment]
https://github.com/techsavvyash/stencil-cli/assets/91770496/dbf7f207-572d-4e99-be51-b9285fe92f63

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@nestjs/cli",
-  "version": "10.1.18",
-  "description": "Nest - modern, fast, powerful node.js web framework (@cli)",
+  "name": "@soorajk1/stencil-cli",
+  "version": "1.1.18",
```
Comment (techsavvyash) [inline package.json]: make this 0.0.1

File: package.json [hunk 2]
```diff
@@ -30,18 +29,16 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/nestjs/nest-cli.git"
+    "url": "git+https://github.com/sooraj1002/stencil-cli.git"
```
Comment (techsavvyash) [inline package.json]: make this techsavvyash as it will be the base repository that will be moved around to orgs and actively maintained.

File: commands/new.command.ts [hunk 3]
```diff
@@ -45,6 +49,8 @@ export class NewCommand extends AbstractCommand {
           value: command.packageManager,
         });
         options.push({ name: 'collection', value: command.collection });
+        options.push({ name: 'prisma', value: command.prisma });
+        console.log('in the new command');
```
Comment (techsavvyash) [inline commands/new.command.ts]: remove this console.log

File: actions/new.action.ts [hunk 4]
```diff
@@ -152,28 +177,62 @@ const installPackages = async (
...
+    console.error('could not generate the prisma files successfully');
+  }
+};
+
 const askForPackageManager = async (): Promise<Answers> => {
   const questions: Question[] = [
     generateSelect('packageManager')(MESSAGES.PACKAGE_MANAGER_QUESTION)([
       PackageManager.NPM,
       PackageManager.YARN,
-      PackageManager.PNPM,
```
Comment (techsavvyash) [inline actions/new.action.ts]: why is this removed?

**Review by techsavvyash — CHANGES_REQUESTED** [review]

**Comment by techsavvyash** [comment]
@sooraj1002 please do all the requested changes and resolve the conversations so that I can go ahead and merge this one.
Thanks for the contribution, this PR seems promising.

File: actions/new.action.ts (hunk 4 above)
Comment (sooraj1002) [reply actions/new.action.ts]: That was a mistake made while testing out the changes. it has been rectified in the latest commit.

**Review by techsavvyash — APPROVED** [review]
LGTM
